### ISSUE TITLE
Messages always show as delivered a few seconds ago

### ISSUE THREAD

**Issue by gmoran1016** [Smiche/MMM-DiscordWatch#3]
As the title says any message I send displays as sent a few seconds ago no matter how long it's been since it was sent. This isn't a huge issue and doesn't really affect my ability to use the module. The module is great by the way!

**Comment by Smiche** [comment]
Ah, that makes sense. A timer is never added to update the time on the messages. It's an easy fix. I can't promise when I'll be able to do it though.

**Comment by gmoran1016** [comment]
Yeah definitely no rush! Just wanted to bring it to you attention.
